### ISSUE TITLE
Game restarts on opposite direction key press

### ISSUE THREAD

**Issue by shaharyar123** [AhsanAyaz/html5-canvas-snake-game#1]
Game restarts 
` if snake is moving on **right direction** and i press **left direction key**`

**Comment by AhsanAyaz** [comment]
@shaharyar123 yeah that's the known issue left as a challenge for the attendees.
I'll let that stay in the source code ;)

**Comment by shaharyar123** [comment]
@AhsanAyaz Got it :smile:
